Mik delft (#37)
open-make/openmakewebsite: content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md

The Microstimulation setup allows for the injection of an electrode, the delivery of an electrical current into a rodent brain, the recording of the cortical response to that stimulus, as well as the monitoring of behaviour (licking, whisking & pupil dilation). 

<img src="https://i.imgur.com/dlNRHpT.png" width="100%" alt="Image of the microstimulation setup, paradigm and behavioural detection, by Mik Schutte.">

*Image of the setup, paradigm and behavioural detection, by Mik Schutte.*
* Main project website: https://github.com/open-make/mik-delft-microstim
* Project (re-)start: 2022
* Core development team size: 2

## Hardware products
The Microstimulation setup is a customisable, open-source experimental setup that can be used for neuroscientific experiments. It uses a micromanipulator, thungsten electrode, stimulus isolator, piezo-sensor, two high-speed cameras and a BPOD.

## Hardware maturity
The setup is roughly 80% of the intended product, for use in our lab. With regards to performance the setup is ready for use. We are adapting the BPOD protocol for online visualisation of results and implementing neurophysiological recordings with multiple-channel probes.

## Rebuilds 

> This project was indeed a rebuilt project from an older setup

Author: ```suggestion
This project was indeed a rebuilt project from an older setup, while we also modified it quite extensively as it is using a different software
```

# Project history

The Microstimulation setup had been developed in the lab mostly by Guy Doron, its use led to two important publication for the lab
 [(Doron et al., 2020](https://www.science.org/doi/10.1126/science.aaz3136); [Shin et al., 2021)](https://www.science.org/doi/10.1126/science.abk1859).
 
 {{< /card >}}

> # Delft Open Hardware Academy

Reviewer: https://hackmd.io/ya-g7omdSI--Vx0JcayB1Q?view

One of the challenges of open hardware documentation is keeping documentation up-to-date as the hardware system evolves. Hardware systems can be complex and dynamic, and it is essential to ensure that documentation reflects the latest changes and updates. One approach to addressing this challenge is to use version control tools, such as GitHub as applied to the microstimulation setup, to track changes to hardware designs and documentation. Version control can help ensure that changes are properly tracked, and it can make it easier to collaborate with others on hardware development.
Another important aspect of open hardware documentation is licensing. By choosing an appropriate license for hardware designs and documentation, designers can ensure that their work is shared and used in the way they intended. The Delft Open Hardware Academy’s lesson on licensing covers everything you need to know for deciding which license is best for your project.

The Academy however, is only a small part of the open hardware community in Delft.

# Delft Open Hardware community

> The existing community of Delft Open Hardware is surrounded by people that are also working on optimizing their documentation for third part use. From this surrounding it is easy to obtain tips and ideas. Moreover, having a community that quickly provides feedback on the comprehensiveness and completeness of the documentation is crucial as it optimizes your work to many different professional backgrounds. Naturally, Delft’s highly diverse community, involving engineers, businesses, biologists, artists, and creative hobbyists fosters an environment for innovation. For instance, I met a developer whom decided to build an anti-theft system to prevent their bike from being stolen. This phone-size device is sensitive to vibrations, which can trigger the alarm to go off. This prompted me to determine if their hardware was also able to detect animal licks for my setup. The same technology is already implemented in some lick-detectors but have high financial cost. Additionally, the Delft community introduced me to a preliminary method for replacing the microstimulation setup’s most expensive component: the micromanipulator. The Open Hardware Automated microscope uses a lower-cost alternative to manipulate a platform, which could be adapted for microstimulation. By making hardware designs and documentation available to a wider audience, open hardware can foster a community of developers and enthusiasts who can work together to improve and enhance hardware systems.

Reviewer: Anti-bike theft: https://www.tudelft.nl/open-hardware/projects/anti-bike-theft-device
Automated microscope: https://www.tudelft.nl/open-hardware/projects/automated-microscope

Author: ```suggestion
The existing community of Delft Open Hardware is surrounded by people that are also working on optimizing their documentation for third part use. From this surrounding it is easy to obtain tips and ideas. Moreover, having a community that quickly provides feedback on the comprehensiveness and completeness of the documentation is crucial as it optimizes your work to many different professional backgrounds. Naturally, Delft’s highly diverse community, involving engineers, businesses, biologists, artists, and creative hobbyists fosters an environment for innovation. For instance, I met a developer whom decided to build an anti-theft system to [prevent their bike from being stolen](https://www.tudelft.nl/open-hardware/projects/anti-bike-theft-device). This phone-size device is sensitive to vibrations, which can trigger the alarm to go off. This prompted me to determine if their hardware was also able to detect animal licks for my setup. The same technology is already implemented in some lick-detectors but have high financial cost. Additionally, the Delft community introduced me to a preliminary method for replacing the microstimulation setup’s most expensive component: the micromanipulator. The [Open Hardware Automated microscope](https://www.tudelft.nl/open-hardware/projects/automated-microscope) uses a lower-cost alternative to manipulate a platform, which could be adapted for microstimulation. By making hardware designs and documentation available to a wider audience, open hardware can foster a community of developers and enthusiasts who can work together to improve and enhance hardware systems.
```

{{< card2 "OSH for the world">}} 

Finally, I learned that open hardware documentation is not just important for hobbyists and enthusiasts. Open hardware documentation can also benefit commercial and industrial applications by enabling faster prototyping, reducing development costs, and facilitating innovation. By making hardware designs and documentation available to a wider audience, companies can tap into a diverse pool of talent and expertise and create more innovative and sustainable hardware products.

{{< /card2 >}}

#  Hardware work

## Berlin time

> In November 2021, I started his master project that would use the microstimulation setup. However, the setup was no longer available in the lab. The people who worked on that paradigm had left the lab, too. Thus, I set out to re-establish the setup. My only source of information was an incomplete wiring diagram. Luckily, with the help of colleagues, I was able to get the basics of the setup to work: stimulate an electrode and detect licks. Next, I implemented the behavioural aquisition, meaning that I set up a camera that were able to clearly capture individual whisker movement and a camera that clearly captured the pupil of the animal. 

Reviewer: In all honesty, Moritz was there to help a lot and he worked with the paradigm before maybe we should mention him explicitly.

Author: ```suggestion
In November 2021, I started his master project that would use the microstimulation setup. However, the setup was no longer available in the lab. The people who worked on that paradigm had left the lab, too. Thus, I set out to re-establish the setup. My only source of information was an incomplete wiring diagram. Luckily, Moritz Drüke knew a lot about the setup and could help me. With his help, I was able to get the basics of the setup to work: stimulate an electrode and detect licks. Next, I implemented the behavioural aquisition, meaning that I set up a camera that were able to clearly capture individual whisker movement and a camera that clearly captured the pupil of the animal. 
```

During this time, Dr. Julien Colomb from the Open.Make team asked me if we could work on making a good documentation of the setup. This would entail a documentation of the hardware, description of the protocol, redesigning the analysis software and getting away from large pay-walls.
As I already was keeping track of the hardware and writing analysis, I gladly agreed.

At the end of my master, I joined the open make project and went to Delft and continue to work on the microstimulation setup and its documentation. The goal was to build 2 setups (I would build one there, Jelte de Vries, another student, would build one in Berlin).

> ## Deflt time

Author: ```suggestion
## Delft time
```

# Successes and failures
**Success :**
Making the microstimulation project open source with regards to both soft- and hardware has already facilitated information transfer between colleagues. For instance, the build-instructions have been utilized to re-build the setup at the lab in my absence. Additionally, information is currently helping a collaboration with [the Gilad lab](https://medicine.ekmd.huji.ac.il/en/research/arielgi/Pages/aboutTheLab.aspx) located in Jerusalem. Finally, our analysis software and accompanying explaination has been used by students in the lab. 

**Fails :**
In the past we have had problems with the ordering of components. This led to long delays which, in turn, caused us to be less productive during a colaboration with the [Delft Open Hardware Academy](https://www.openhardware.academy/01_Welcome.html). We also have have lost some time while trying to develop the BPOD laboratory interface in Python. [PyBPOD](https://pybpod.readthedocs.io/projects/pybpod-api/en/v1.8.1/) is a suitable product, yet I would discourage it's use for new users.

> 

Author: ## I am a better researcher

The fellowship has been instrumental in shaping my career profile and career path. Through hands-on experience, I acquired valuable skills in soldering, PCB design, 3D printing, and project management. These technical skills have enhanced my competence as a scientist and are actively being put to use. For instance, I have recently designed a PCB layout for microstimulation; the Arduino configures the waveform and soldered wires send this to the stimulus isolator with everything nicely housed in its 3D-printed ‘containers’.  Moreover, the exposure to project management has honed my ability to plan, organize, and execute tasks efficiently. Overall, the fellowship has made me into a more well-rounded researcher and has feeling well-prepared for the coming years of my PhD.

> 

Author: ```suggestion
## I am a better researcher

The fellowship has been instrumental in shaping my career profile and career path. Through hands-on experience, I acquired valuable skills in soldering, PCB design, 3D printing, and project management. These technical skills have enhanced my competence as a scientist and are actively being put to use. For instance, I have recently designed a PCB layout for microstimulation; the Arduino configures the waveform and soldered wires send this to the stimulus isolator with everything nicely housed in its 3D-printed ‘containers’.  Moreover, the exposure to project management has honed my ability to plan, organize, and execute tasks efficiently. Overall, the fellowship has made me into a more well-rounded researcher and has feeling well-prepared for the coming years of my PhD.
```

> # Aknowledgements

Reviewer: Acknowledgements*

Author: ```suggestion
# Acknowledgements
```
